apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -26,11 +26,13 @@ const SongSelector = Radium(class extends React.Component {
     setSong: PropTypes.func.isRequired,
     selectedSong: PropTypes.string.isRequired,
     songManifest: PropTypes.arrayOf(PropTypes.object).isRequired,
-    hasChannel: PropTypes.bool.isRequired
+    hasChannel: PropTypes.bool.isRequired,
+    userType: PropTypes.string.isRequired
   };
 
   state = {
-    songsData: []
+    songsData: [],
+    filterOff: this.props.userType !== 'student_y'
```

Comment:
Nit: You only set `filterOff` when constructing the component, so it could be a prop rather than React state (you'd move this expression into your `connect()` method).